MQTT5-UserGuide.md

```diff
@@ -298,6 +299,23 @@ A MQTT5 direct connection can be made using a PKCS12 file rather than using a PE
 
 Note: Currently, TLS integration with PKCS#12 is only available on MacOS devices.
 
+#### Direct MQTT with Windows Certificate Store Method
+
+A MQTT5 direct connection can be made with mutual TLS with the certificate and private key in the Windows certificate store,
+rather than simply being files on disk. To create a MQTT5 builder configured for this connection, see the following code:
+
+```typescript
+    // Certificate store path below is an example.
+    let certificateStorePath : string = "CurrentUser\\MY\\A11F8A9B5DF5B98BA3508FBCA575D09570E0D2C6";
```

Comment:
Yes, this path is just a reference on a local machine, we use it in the Java migration guide too. I thought it might be helpful for users to see how the parameter should look like.